src/components/Display.js

```diff
@@ -0,0 +1,122 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { Link } from 'react-router-dom';
+import { CircularProgressbar } from 'react-circular-progressbar';
+import 'react-circular-progressbar/dist/styles.css';
+import { removeBookSuccess, fetchBooks } from '../redux/books/books';
+import '../index.css';
+
+const Books = () => {
+  const dispatch = useDispatch();
+  const booksStatus = useSelector((state) => state.books.loading);
+
+  useEffect(() => {
+    if (booksStatus === false) {
+      dispatch(fetchBooks());
+    }
+  }, []);
+
+  const removeBookBtn = (e) => {
+    dispatch(removeBookSuccess(e.target));
+  };
+
+  const myBooks = useSelector((state) => state.books.books);
+
+  const AddBookCard = () => (
+    <div>
+      {myBooks.map((book) => {
+        const randomNum = Math.floor(Math.random() * 100);
+        const barstyle = {
+          width: `${randomNum}%`,
+        };
+
+        return (
+          <div className="card" key={book.id}>
```

Comment:
- [ OPTIONAL]
- You will notice that each book card has a margin between them but since this was accidentally skipped by the previous reviewer, I will mark it as optional but it would be great if you implement it.
- 
![image](https://user-images.githubusercontent.com/68167810/132100589-0f30f96a-477f-47b7-bf69-d4462f970dfa.png)

- [ OPTIONAL]
- Kindly remove the horizontal scrolling on your page to give it a better user experrience.